lms/djangoapps/certificates/views/webview.py

```diff
@@ -425,10 +444,11 @@ def _update_badge_context(context, course, user):
     """
     Updates context with badge info.
     """
-    try:
-        badge = BadgeAssertion.objects.get(user=user, course_id=course.location.course_key)
-    except BadgeAssertion.DoesNotExist:
-        badge = None
+    badge = None
+    if settings.FEATURES.get('ENABLE_OPENBADGES') and course.issue_badges:
```

Comment:
hide behind "is_feature_enabled" method -- now it seems 'is_badging_enabled' might be a better name